CONTRIBUTING.md

```diff
@@ -194,6 +194,9 @@ instructions are below in case you need them.
    4. Click "Browse", and navigate to the file `build-conventions/formatterConfig.xml`
    5. **IMPORTANT** - make sure "Optimize Imports" is **NOT** selected.
    6. Click "OK"
+   7. Optional: If you like to format code changes on save automatically, open
+      **Preferences > Tools > Actions on Save** and check "Reformat Code" with the additional
+      options of your choise (e.g. select "Changed lines" to only reformat changed code)
```

Comment:
I don't think we want to suggest only formatting changed lines, as it gives the wrong impression of what the formatting requirements are. I'd just omit that part:

```suggestion
      **Preferences > Tools > Actions on Save** and check "Reformat Code", making sure to
      configure Java files.
```